client/components/ScenariosList/index.jsx

```diff
@@ -7,46 +7,49 @@ import PropTypes from 'prop-types';
 import 'semantic-ui-css/semantic.min.css';
 import './ScenariosList.css';
 
-const ScenarioEntries = ({ scenarioData, isLoggedIn }) => {
+const ScenarioEntries = ({ scenarioData, isLoggedIn, path }) => {
     if (!scenarioData.length) {
         return null;
     }
 
-    return scenarioData.map(({ id, title, description }) => {
-        return (
-            <Grid.Column
-                mobile={16}
-                tablet={8}
-                computer={5}
-                key={id}
-                className="tm__scenario-entry"
-            >
-                <Header as="h3">
-                    <Link
-                        to={{
-                            pathname: `/moment/${id}`,
-                            state: { id, title, description }
-                        }}
-                    >
-                        {title}
-                    </Link>
-                </Header>
-                <p className="tm__scenario-desc">{description}</p>
-                {isLoggedIn && (
-                    <Button
-                        basic
-                        fluid
-                        color="black"
-                        push="true"
-                        as={Link}
-                        to={{ pathname: `/editor/${id}` }}
-                    >
-                        Edit
-                    </Button>
-                )}
-            </Grid.Column>
-        );
-    });
+    // only filter out scenario if there is a path
+    return scenarioData
+        .filter(d => (!path && d) || d.categories.includes(path))
```

Comment:
Nitpick: can we name this variable something more descriptive than `d`? I understood what it was, but it took me a moment.